components/src/Field/Field.js

```diff
@@ -25,6 +26,9 @@ const Field = ({
   </Label>
 );
 
+const Field = styled(BaseField)`
+`;
```

Comment:
The template literal could go on the same line: ` const Field = styled(BaseField)`` `